backend/hct_mis_api/apps/payment/mutations.py

```diff
@@ -951,6 +952,13 @@ def mutate(cls, root: Any, info: Any, input: Dict, **kwargs: Any) -> "AssignFspT
         if len(mappings) != payment_plan.delivery_mechanisms.count():
             raise GraphQLError("Please assign FSP to all delivery mechanisms before moving to next step")
 
+        if payment_plan.currency == USDC:
```

Comment:
also maybe dropdown choices should be limited only to DELIVERY_TYPE_TRANSFER_TO_DIGITAL_WALLET

Reply:
not sure about limit only for `DELIVERY_TYPE_TRANSFER_TO_DIGITAL_WALLET`
because in the task we just have next:
> Alert the user if a digital currency was initially selected, and the delivery mechanism was a non-digital one. (i.e., Alert user if USDC was selected to 'Cash by FSP' mechanism)